.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Upload files to memecentral
+        uses: appleboy/scp-action@master
+        env:
+          HOST: ${{ secrets.HOST }}
+          USERNAME: ${{ secrets.USERNAME }}
+          KEY: ${{ secrets.KEY }}
+        with:
+          source: '.'
+          target: '/home/memebot'
+      - name: Reload PM2 server
+        uses: appleboy/ssh-action@master
+        env:
+          MONGO_URL: ${{ secrets.MONGO_URL }}
+        with:
+          host: ${{ secrets.HOST }}
+          username: ${{ secrets.USERNAME }}
+          key: ${{ secrets.KEY }}
+          env: MONGO_URL
```

Comment:
Why is MONGO_URL set here but the other env variables set in the PM2 config file?

Reply:
It contains the private IP of the database server and I didn't wanna expose it, not that it would be a huge deal.

Reply:
Oh okay, I'm getting confused with the 

`with:
     env: MONGO_URL`

vs. 

`env:
    MONGO_URL: ...`

What does the with statement do?

Reply:
This might help [ssh-remote-commands](https://github.com/marketplace/actions/ssh-remote-commands). Basically the first `env` is for GitHub Actions to declare the env vars just for that step, and then the second is for the custom action which copies the env var names in `env` to the ssh shell.

Reply:
Lol I misspelled it